### PR TITLE
fix: code correction and rephrasing on optimization page

### PR DESCRIPTION
**Pull request by jeremenichelli**
I was late to the PR that added the page so I'm proposing some changes here. thanks @EugeneHlushko for adding it 😊

### CODE REVIEW

File: src/content/configuration/optimization.md [hunk 1]
```diff
@@ -3,17 +3,22 @@ title: Optimization
...
 contributors:
   - EugeneHlushko
+  - jeremenichelli
+related:
+  - title: 'webpack 4: Code Splitting, chunk graph and the splitChunks optimization'
+    url: https://medium.com/webpack/webpack-4-code-splitting-chunk-graph-and-the-splitchunks-optimization-be739a861366
 ---
 
-Optimization configuration allows you to control the webpack output optimization level.
+Since version 4 webpack runs optimizations for you depending on the `mode` set, but you can modify them indepentently too.
```
Comment (EugeneHlushko) [inline src/content/configuration/optimization.md]: Maybe say:
- ...depending on the  *chosen* `mode` or `mode` *selected*. Word *set* here is a bit confusing to me.
- ...but you can modify them indepentently too. -> All optimizations are available for manual configuration and overrides.

File: src/content/configuration/optimization.md [hunk 2]
```diff
@@ -22,16 +27,18 @@ __webpack.config.js__
...
 ```
 
+T> Learn how [mode](/concepts/mode/) works.
+
 ## `optimization.minimizer`
 
 `UglifyjsWebpackPlugin | [UglifyjsWebpackPlugin]`
 
-Allows to override default minimizer by providing one or more custom configured [UglifyjsWebpackPlugin](/plugins/uglifyjs-webpack-plugin/) instance(s).
+Allows you to override the default minimizer by providing a different one or more customized [UglifyjsWebpackPlugin](/plugins/uglifyjs-webpack-plugin/) instance.
```
Comment (EugeneHlushko) [inline src/content/configuration/optimization.md]: one or more isntance**s** ?

**Comment by jeremenichelli** [comment]
Done @EugeneHlushko, let me know how it looks!

File: src/content/configuration/optimization.md [hunk 3]
```diff
@@ -41,7 +48,7 @@ module.exports = {
   //...
   optimization: {
     minimizer: [
-      new UglifyJsPlugin({ /* … */ })
+      new webpack.optimize.UglifyJsPlugin({ /* your config */ })
```
Comment (montogeek) [inline src/content/configuration/optimization.md]: Ins't this duplicate?

Comment (jeremenichelli) [reply src/content/configuration/optimization.md]: I that's true, I'll fix it.

**Pull request merged**